editor/components/block-list/block.js

```diff
@@ -502,6 +503,7 @@ export class BlockListBlock extends Component {
 						onDragEnd={ this.onDragEnd }
 						isDragging={ dragging }
 						elementId={ blockElementId }
+						draggable={ isMovable } // only in locking all moving blocks is totally impossible
```

Comment:
What does this comment mean? I don't quite understand it.

Reply:
The main reason is that we depend on the styles set by BlockDraggable. Without BlockDraggable being rendered, it becomes almost impossible to get the delete and ellipsis buttons to appear. When multi-selecting it is not a problem because they appear there all the time even if the mouse is not close to the blocks.
I will try to apply a refactor to the styles.

Reply:
> I will try to apply a refactor to the styles.

That would be awesome, thanks for looking into it 👍